src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -223,27 +261,14 @@ const formConfig = {
         page4: {
           path: 'signer-contact-info',
           title: 'Certification',
-          uiSchema: {
-            ...titleUI(
-              'Your contact information',
-              'We use this information to contact you and verify other details.',
-            ),
-            certifierPhone: phoneUI(),
-            certifierEmail: emailUI(),
-          },
-          schema: {
-            type: 'object',
-            required: ['certifierPhone', 'certifierEmail'],
-            properties: {
-              titleSchema,
-              certifierPhone: phoneSchema,
-              certifierEmail: emailSchema,
-            },
-          },
+          CustomPage: SignerContactInfoPage,
+          CustomPageReview: null,
+          ...signerContactInfoPage,
```

Comment:
Switched to a custom page here so that we can do a custom `onGoForward` function. That custom function is what copies the certifier information to the appropriate other locations in the form.